app/assets/stylesheets/_base.scss

```diff
@@ -6,3 +6,13 @@ html{
 body{
   padding: 100px 20px 100px 20px;
 }
+
+hr{
+  border-top: 1px solid #D4D4D4;
+}
+
+.application-show .section{
+  padding: 20px;
+  border: 1px solid #A2A2A2;
```

Comment:
Color literals like `#A2A2A2` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#A2A2A2` should be written as `#a2a2a2`